monkey/infection_monkey/i_puppet/payload_result_data.py

```diff
@@ -0,0 +1,7 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class PayloadResultData:
```

Comment:
What about 
```suggestion
class PayloadResult:
```

Reply:
I don't mind, but we have `ExploiterResultData`, so `PayloadResultData` would be more consistent.

Reply:
Ok. I think we should change them both. Followup PR? It requires rebuilding plugins (probably).

Reply:
Changed `PayloadResultData`, opened #3532 for `ExploiterResultData`